python_modules/libraries/dagster-aws/dagster_aws_tests/ecs_tests/launcher_tests/test_launching.py

```diff
@@ -330,6 +331,15 @@ def test_reuse_task_definition(instance, ecs):
         )
     )
 
+    # Changed command fails
```

Comment:
I don't think our sidecar comparison is directly unit tested either.

Reply:
it is now!